style.css

```diff
@@ -0,0 +1,135 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+#header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.my-logo {
+  text-decoration: none;
+  color: #fff;
+  font-family: 'Roboto', sans-serif;
+}
+
+body {
+  background: #1c1a19;
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 24px;
+  gap: 12px;
+  position: absolute;
+  width: 375px;
+  height: 590px;
+  left: 0;
+  top: 92px;
+}
+
+.h1 {
+  display: flex;
+  justify-content: space-between;
+  height: 124px;
+  width: 375px;
+  left: 24px;
+  top: 120px;
+  font-family: 'Crete Round', serif;
+  color: #ff6b00;
+  font-weight: 400;
+  font-size: 56px;
+  letter-spacing: 0.37px;
+  margin: 10px 0;
+}
+
+.nav {
+  background: #3c3a39;
+  background-size: 200%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 8px 20px;
+  position: absolute;
+  width: 375px;
+  height: 92px;
+  left: 0;
+  top: 0;
+  font-family: 'Roboto', sans-serif;
+  color: #fff;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+}
+
+.h2-description {
+  width: 375px;
+  top: 256px;
+  left: 24px;
+  font-family: 'Roboto', sans-serif;
+  color: #fff5e1;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
```

Comment:
- [ ] Please ensure that the fonts are as close to pixel-perfect as possible. Pay attention to details and make necessary adjustments to achieve precise alignment and consistency in font rendering.

- [ ] Yours
![image](https://github.com/valentino2105/Portfolio-setup-and-mobile-first/assets/68823331/9b94e190-8827-416d-92bf-64079cb41f60)

- [ ] Figma
![image](https://github.com/valentino2105/Portfolio-setup-and-mobile-first/assets/68823331/c80195bc-f683-4631-a5ac-bf9354408c27)